### PR TITLE
Webdevtoday/html movie seat booking

### PR DESCRIPTION
**Pull request by webdevtoday**
Changed a view of focused state of checkboxes.
[Demo](https://webdevtoday.github.io/html-movie-seat-booking/)

### CODE REVIEW

**Comment by A-Ostrovnyy** [comment]
@webdevtoday Please provide us with a link to the working demo.

File: submissions/webdevtoday/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Task</title>
```
Comment (A-Ostrovnyy) [inline submissions/webdevtoday/html-movie-seat-booking/index.html]: Add appropriate title

File: submissions/webdevtoday/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                                        <span class="Seat_view"></span>
+                                    </label>
+                                    <label class="Form_label Seat">
+                                        <input type="checkbox" name="seats" value="62" class="Seat_checkbox" disabled>
+                                        <span class="Seat_view"></span>
+                                    </label>
+                                </div>
+
+                            </div>
+
```
Comment (A-Ostrovnyy) [inline submissions/webdevtoday/html-movie-seat-booking/index.html]: Remove unused lines.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]

File: submissions/webdevtoday/html-movie-seat-booking/index.html (hunk 1 above)
Comment (webdevtoday) [reply submissions/webdevtoday/html-movie-seat-booking/index.html]: I'll fix it

File: submissions/webdevtoday/html-movie-seat-booking/index.html (hunk 2 above)
Comment (webdevtoday) [reply submissions/webdevtoday/html-movie-seat-booking/index.html]: I'll fix it

File: submissions/webdevtoday/html-movie-seat-booking/index.html (hunk 1 above)
Comment (webdevtoday) [reply submissions/webdevtoday/html-movie-seat-booking/index.html]: I've done it

File: submissions/webdevtoday/html-movie-seat-booking/index.html (hunk 2 above)
Comment (webdevtoday) [reply submissions/webdevtoday/html-movie-seat-booking/index.html]: I've done it

**Comment by A-Ostrovnyy** [comment]
@webdevtoday can you add a link on the demo page?)

**Comment by webdevtoday** [comment]
> @webdevtoday can you add a link on the demo page?)

Oh, sorry, I'll add it

**Comment by webdevtoday** [comment]
> @webdevtoday can you add a link on the demo page?)

I've done it

**Review by A-Ostrovnyy — APPROVED** [review]
Good Work!

**Pull request merged**